### PR TITLE
Use helper function from library-go for config check

### PR DESCRIPTION
**Pull request by damemi**

### CODE REVIEW

**Comment by damemi** [comment]
cc @deads2k

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -2,21 +2,18 @@ package operator
...
-	"os"
 	"time"
 
 	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/apis/kubecontrollermanager/v1alpha1"
 	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/v311_00_assets"
-	"k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/runtime/serializer"
+	"github.com/openshift/library-go/pkg/operator/v1alpha1helpers"
```
Comment (deads2k) [inline pkg/operator/starter.go]: nit, order

File: pkg/operator/starter.go [hunk 2]
```diff
@@ -29,6 +26,10 @@ func RunOperator(clientConfig *rest.Config, stopCh <-chan struct{}) error {
 	if err != nil {
 		panic(err)
```
Comment (deads2k) [inline pkg/operator/starter.go]: fix this while you're here.

**Comment by damemi** [comment]
@deads2k updated

**Comment by sttts** [comment]
/lgtm
/approve

what's up with prow?

**Pull request merged**